src/main/java/org/openrewrite/java/template/internal/TemplateCode.java

```diff
@@ -37,7 +37,10 @@ public static <T extends JCTree> String process(T tree, List<JCTree.JCVariableDe
             StringBuilder builder = new StringBuilder("JavaTemplate\n");
             builder
                     .append("    .builder(\"")
-                    .append(writer.toString().replace("\\", "\\\\").replace("\"", "\\\""))
+                    .append(writer.toString()
+                            .replace("\\", "\\\\")
+                            .replace("\"", "\\\"")
+                            .replaceAll("\\R", " "))
```

Comment:
I would rather replace them with an escaped line terminator (`\n`) because in a text block they are significant whitespace.

Reply:
Better indeed; done in https://github.com/openrewrite/rewrite-templating/pull/62/commits/367c450e1eb19d77182b3fde965763f382b93b92